lib/src/bazelify/pubspec.dart

```diff
@@ -67,4 +85,22 @@ class Pubspec {
 
   /// Name of the package.
   String get pubPackageName => _pubspecContents['name'];
+
+  List<Transformer> _transformers;
```

Comment:
Sure, I would actually move all the getters above as well but I don't think there is any consensus around that really so I won't touch things.